playbooks/templates/common/metadata_common.yaml

```diff
@@ -0,0 +1,10 @@
+metadata          :
+  RPC: {{ maas_rpc | default(true) }}
+  RPC_env_identifier: {{ maas_env_identifier | default('unkown')}}
+  RPC_env_identifier_alt: {{ maas_env_identifier_alt | default('unkown')}}
+  RPC_maas_version: {{  lookup('pipe', 'cd .. && git describe --tags --abbrev=0') }}
+  RPC_deploy_date: {{ ansible_date_time.date }}
+  RPC_check_category: {{ check_category | default('Unkown') }}
```

Comment:
I'm not seeing where this is set. It would have to be defined in each template, correct? We probably want to prepend `maas_` to the variable name.

Reply:
Done.